requirements/requirements_multimodal.txt

```diff
@@ -0,0 +1,20 @@
+addict
+basicsr
+clip
+diffusers>=0.19.3
+einops_exts
+# webdataset>=0.2.48 conflict with ASR models, manually pip install when using multimodal
+# flash-attn>=2.0.1
+imageio-ffmpeg
+kornia
+nerfacc>=0.5.3
+open_clip_torch
+opencv-python>=4.8.0.74
+PyMCubes
+pymeshlab
```

Comment:
License is GNU v3 self replicating license, which is not allowed in NeMo. Code - we only allow Apache 2, MIT, BSD 3 - please discuss with @ericharper 
https://github.com/cnr-isti-vclab/PyMeshLab/blob/main/LICENSE

Reply:
yeah, we are checking this.